examples/js/animation/MMDAnimationHelper.js

```diff
@@ -307,6 +307,7 @@ THREE.MMDAnimationHelper = ( function () {
 			this.objects.set( mesh, { looped: false } );
 
 			this._setupMeshAnimation( mesh, params.animation );
+			this._setupBoneOriginal( mesh );
```

Comment:
Original Bone transformations need to be saved in `pose()`, too, because meshes for `pose()` don't need to be added to the helper beforehand.

Reply:
Can I get original position from `` pose() `` ?
I not sure I understand what you said.